.pre-commit-config.yaml

```diff
@@ -20,3 +20,7 @@ repos:
      -  id: mypy
         files: sklearn/
         additional_dependencies: [pytest==6.2.4]
+-   repo: https://github.com/MarcoGorelli/cython-lint
+    rev: v0.14.0
+    hooks:
+    -   id: cython-lint
```

Comment:
There's another tool in cython-lint named ``double-quote-cython-strings``. It changes all single quote string into double quotes (like black). I did not add it for now (mainly because you can't do ``double-quote-cython-strings .`` yet). Leaving that for a subsequent PR